Dockerfile

```
@@ -1,11 +1,11 @@
-FROM node:18-bullseye as base
+FROM node:18-bookworm as base
 RUN apt-get update && apt-get install -y openssl
 WORKDIR /app
 ADD .yarn ./.yarn
```

Hadolint

[error] Use COPY instead of ADD for files and folders

(DL3020)